clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -4096,6 +4114,15 @@ RValue CodeGenFunction::EmitBuiltinExpr(const GlobalDecl GD, unsigned BuiltinID,
   case Builtin::BI__builtin_frexpf128:
   case Builtin::BI__builtin_frexpf16:
     return RValue::get(emitFrexpBuiltin(*this, E, Intrinsic::frexp));
+  case Builtin::BImodf:
+  case Builtin::BImodff:
+  case Builtin::BImodfl:
+  case Builtin::BI__builtin_modf:
+  case Builtin::BI__builtin_modff:
+  case Builtin::BI__builtin_modfl:
+    if (Builder.getIsFPConstrained())
```

Comment:
Don't think this strictfp handling is tested?

Reply:
`math-libcalls.c` sets `-ffp-exception-behavior=maytrap` for the `MAY_TRAP` tests, which hit this. Is there another way to test this?